README.md

```diff
@@ -1,269 +1,44 @@
 # Ballerina Twitter Connector
-Connects to Twitter using Ballerina.
 
-# Introduction
-The Twitter API can be used to programmatically retrieve and analyze data, as well as engage with the conversation on Twitter.
-This connector provides access to a variety of different resources in Twitter including the following:
-* Tweets
-* Users
- 
-The Twitter API currently consists of two supported versions, as well as different access tiers.
-When user wants to access Twitter through Ballerina Twitter Connector, they are required to register an application. By default, applications can only access public information on Twitter. Certain operations, such as those responsible for sending or receiving Direct Messages and posting a Tweet, require additional permissions from user before access user information. These permissions are not granted by default; you choose on a per-application basis whether to provide this access, and can control all the applications authorized on your account.
+[![Build](https://github.com/ballerina-platform/module-ballerinax-twitter/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-twitter/actions?query=workflow%3ACI)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-twitter.svg)](https://github.com/ballerina-platform/module-ballerinax-twitter/commits/master)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## What is Twitter?
-Twitter is a social networking service on which users can post and communicate with status called as "Tweets". Authenticated users can post, like, and retweet tweets.  Main use case is to connect people and allow people to express their thoughts to large set of people.
+[Twitter](https://twitter.com/) is a social media service provided by Twitter.
 
-## Key Features of Twitter
-* Create and manage Tweet, Retweet and Reply
-* Manage Users and their details
+This connector provides operations for connecting and interacting with Twitter endpoints over the network. 
+For more information about configuration and operations, go to the module(s).
+- [`twitter`](twitter/Module.md)
 
-# Supported Versions & Limitations
+## Building from the source
+### Setting up the prerequisites
 
-## Supported Versions
+1. Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
 
-|                                   | Version               |
-|:---------------------------------:|:---------------------:|
-|       Twitter API Version         |         v1.1          |
-|       Ballerina Language          |   Swan Lake Alpha 5   |
+   > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
```

Comment:
Use markdown syntax for note, if cannot use this as a step.

Reply:
This is how it is across all the connectors